### ISSUE TITLE
MMP Atlas

### ISSUE THREAD

**Issue by MrMercier** [fieldtrip/fieldtrip#2103]
Hi Fieldtrip Team,

I was looking for the so-called "Glasser" Atlas (DOI: 10.1038/nature18933, see description [webpage](https://balsa.wustl.edu/study/show/RVVG)) and thanks to the ft_mailinglist, I found that the file is:
- referred [there](https://www.fieldtriptoolbox.org/tutorial/networkanalysis/)
- downloadable [here](https://download.fieldtriptoolbox.org/tutorial/networkanalysis/).

The structure is quite clear and I am wondering why it is not on the [ft_webpage dedicated to the Atlases](https://www.fieldtriptoolbox.org/template/atlas/)?

Would it be possible to add it ? Can I give a helping hand ? (e.g. writing the documentation).

Kindly,

Manuel

**Comment by schoffelen** [comment]
Hi Manuel @MrMercier !

I must say that I am (in general) a bit reluctant to share (more visibly) atlases that are defined on the cortical surface. Most important reason for this is that it is not fully trivial to get the subject-specific sourcemodel that is in register with the specific atlas. If we would include the atlas that is used in the tutorial in the atlas folder, we also would need to explain how to generate the cortical surface that goes with this atlas. 

Would it be a good compromise to write a little bit of documentation, that links to an URL from where the atlas can be downloaded + add a few cautionary words about what to keep in mind if such an atlas is to be used? e.g. https://balsa.wustl.edu/sceneFile/x2XDx and there is probably also a volumetric version of this atlas available for download.

**Comment by schoffelen** [comment]
Hi Manuel, @MrMercier what shall we do with this issue? We don't have the resources at the moment to push this forward.

**Comment by MrMercier** [comment]
Hi Jan-Mathijs,

I finally gathered the following information (btw, even registered I was not allowed to access the link you shared, you must have some privilege ; ))

There is a shared [HCP-MMP1.0 projected on MNI2009a GM (volumetric) in NIfTI format](https://figshare.com/articles/dataset/HCP-MMP1_0_projected_on_MNI2009a_GM_volumetric_in_NIfTI_format/3501911) (maybe that's the one that ended in the ft_mailing list - see my first post).

Yet, as you pointed out, as this atlas is defined on the cortical surface it is rather recommended to do a subject-specific mapping. For instance by using the [HCP-MMP1.0 projected on fsaverage](https://figshare.com/articles/dataset/HCP-MMP1_0_projected_on_fsaverage/3498446?file=5528837). 

Then the .annot files (surface parcellations) can be projected to the subject space, and the surfaces parcellations interpolated into the volume. The freesurfer commands being:
_mri_surf2surf --srcsubject fsaverage --trgsubject $subj --hemi lh --sval-annot /path/to/lh.HCP-MMP1.annot --tval $sd/label/lh.HCP-MMP1.annot
mri_surf2surf --srcsubject fsaverage --trgsubject $subj --hemi rh --sval-annot /path/to/rh.HCP-MMP1.annot --tval $sd/label/rh.HCP-MMP1.annot
mri_surf2volseg --o $sd/mri/HCP-MMP1+aseg.mgz --label-cortex --i $sd/mri/aseg.mgz --threads 1 --lh-annot $sd/label/lh.HCP-MMP1.annot 0 --lh-cortex-mask $sd/label/lh.cortex.label --lh-white $sd/surf/lh.white --lh-pial $sd/surf/lh.pial --rh-annot $sd/label/rh.HCP-MMP1.annot 200 --rh-cortex-mask $sd/label/rh.cortex.label --rh-white $sd/surf/rh.white --rh-pial $sd/surf/rh.pial_ 
(for the more curious [mri_surf2volseg](https://github.com/freesurfer/freesurfer/blob/dev/mri_aparc2aseg/mri_surf2volseg.help.xml) seems to be a 'new' version of [mri_aparc2aseg](https://github.com/freesurfer/freesurfer/tree/dev/mri_aparc2aseg).

Still, the [original study](https://balsa.wustl.edu/study/show/RVVG) proposes a "mapping machine-learning classifier to recognize the multi-modal ‘fingerprint’ of each cortical area". And the authors "anticipate that the areal classifier will generalize to other studies that acquire the following core set of MRI images: high-resolution T1w and T2w; spin echo-based b0 field map; and extensive fMRI data acquired using ‘multiband’ pulse sequences to improve spatial and temporal resolution".
In other words, what I understand is that to reach an optimal results multimodal data are required and the freesurfer'solution above is an approximation (still better than the use of the volumetric version projected on MNI).

If it helps, I can write a bit of documentation for ft website, but maybe not in this issue because it relates to a more general concern about the use of Atlases ; )


Ps
Thanks to Francois for his explanation on [intranat git](https://github.com/ftadel/IntrAnat/issues/7).
